docs/Publishing.md

```diff
@@ -0,0 +1,28 @@
+# Merging and Publishing
+
+Merging your code means that the features or fix was reviewed and approved by the Design System Team and that a new version is ready to be published!
+When you merge you code you are all done here. A new version of the library will be release as soon as the pipeline finishes.
+
+## Before merging your code
+
+Before opening a merging, please be sure that:
+
+- your code was reviewed and approved by the Design System Team developers
+- your changes were visually validated by the Design System Team designers
+- your changes were approved and the jira issue was moved to waiting release
```

Comment:
Add comment: "Doesn't apply if you are a developer outside Natura organization"

Reply:
boa! done in c5e361b